README.md

```diff
@@ -98,9 +98,25 @@ declaring that, while 4.1.18 and 4.1.20 are in `candidate-4.2`, they should not
 ### Block Edges
 
 Create/edit an appropriate file in `blocked_edges/`.
-- `to` is the release which has the existing incoming edges.
-- `from` is a regex for the previous release versions.
+
+* `to` (required, [string][json-string]) is the release which has the existing incoming edges.
+* `from` (required, [string][json-string]) is a regex for the previous release versions.
     If you want to require `from` to match the full version string (and not just a substring), you must include explicit `^` and `$` anchors.
+    Release version strings will receive [the architecture-suffix](#release-names) before being compared to this regular expression.
+* `url` (optional, [string][json-string]), with a URI documenting the blocking reason.
+    For example, this could link to a bug's impact statement or knowledge-base article.
+* `name` (optional, [string][json-string]), with a CamelCase reason suitable for [a `ClusterOperatorStatusCondition` `reason` property][api-reason].
```

Comment:
Do we need to come up with a name when its tied to a bug is (almost) all cases?

Reply:
This slug [eventually feeds the `Recommended` condition's `reason` property][1].  I think we want that, although see the *Metadata to include when blocking a conditional request* section in the enhancement for discussion.

[1]: https://github.com/openshift/enhancements/pull/821#discussion_r705538121

Reply:
I wonder if we maintain the list of name i.e. `ClusterOperatorStatusCondition` `reason` it might be useful to create some kind of glossary